webapp/channels/src/components/post_emoji/post_emoji.tsx

```diff
@@ -22,14 +24,22 @@ const PostEmoji = ({name, imageUrl}: Props) => {
     }
 
     return (
-        <span
-            alt={emojiText}
-            className='emoticon'
+        <WithTooltip 
```

Comment:
The way this works right now, I worry what happens if there are many emojis in screen. Could you do a quick test for me?

See if there is any visible difference between this PR and master when you have a post with many emojis (copy paste `:+1: ` until you get to the maximum number of characters).

The issue I see is that `WithTooltip` creates a div like this every time it is present:

`<div style="display: flex; flex-direction: column; align-items: center; z-index: 1070; position: absolute; top: -1000px; left: -1000px;" class="tooltip-container"></div>` and if we are rendering a lot of emojis, we may create many of these divs, and I don't know how troublesome for the browser that is.

Reply:
This div will created when hover is active for the emoji. Once hover exited for the element the created div is destroyed from the document.